Source/WebKit/UIProcess/Cocoa/WKTextExtractionUtilities.h

```diff
@@ -38,6 +41,8 @@ struct Item;
 namespace WebKit {
 
 void prepareTextExtractionSupportIfNeeded();
-RetainPtr<WKTextExtractionItem> createItem(const WebCore::TextExtraction::Item&);
+
+using RootViewToWebViewConverter = Function<WebCore::FloatRect(const WebCore::FloatRect&)>;
+RetainPtr<WKTextExtractionItem> createItem(const WebCore::TextExtraction::Item&, const RootViewToWebViewConverter&);
```

Comment:
Nit:
```suggestion
RetainPtr<WKTextExtractionItem> createItem(const WebCore::TextExtraction::Item&, RootViewToWebViewConverter&&);
```

Reply:
Oh, actually, maybe not given how `createItemRecursive` works.

Reply:
Yeah, we could make the first call to `createItem` an rvalue, but then we'll need const ref for the recursive calls.